tests/delete_test.go

```
@@ -418,6 +419,7 @@ func TestHardDeleteAfterSoftDelete(t *testing.T) {
 }
 
 func TestDeleteWithLimitAndOrder(t *testing.T) {
+	DB.Exec(`DROP TABLE "users"`).Commit()
```

Can you add a condition to ignore the error if the table doesn't exist and returns ORA-00942?
```
if err := DB.Exec(`DROP TABLE "users"`).Commit().Error; err != nil {
		if !strings.Contains(err.Error(), "ORA-00942") {
			t.Fatalf("Failed to migrate, got error: %s", err)
		}
}
```